docs/how-to/hip_runtime_api.rst

```diff
@@ -0,0 +1,35 @@
+.. meta::
+  :description: HIP runtime API usage
+  :keywords: AMD, ROCm, HIP, CUDA, HIP runtime API How to,
+
+.. _hip_runtime_api_how-to:
+
+********************************************************************************
+HIP runtime API
+********************************************************************************
+
+The HIP runtime API provides C and C++ functionalities to manage event, stream,
+and memory on GPUs. On the AMD platform, the HIP runtime uses
+:doc:`Common Language Runtime (CLR) <hip:understand/amd_clr>`, while on NVIDIA
```

Comment:
Should be `Compute Language Runtime (CLR)

Reply:
Fixed and updated the link side also.